showcase/app/templates/page-components/rich-tooltip.hbs

```diff
@@ -44,41 +44,60 @@
 
   <Shw::Text::H4>Sizes</Shw::Text::H4>
 
-  {{#let (array "inherited" "small" "medium" "large") as |sizes|}}
-    {{#each sizes as |size|}}
-      <Shw::Flex @label="{{capitalize size}} {{if (eq size 'inherited') '(default)'}}" as |SF|>
-        <SF.Item>
-          <Hds::RichTooltip as |RT|>
-            <RT.Toggle @text="Lorem ipsum dolor" @size={{if (eq size "inherited") undefined size}} />
-            <RT.Bubble>
-              <Shw::Placeholder @text="generic content" @height="40" />
-            </RT.Bubble>
-          </Hds::RichTooltip>
-        </SF.Item>
-        <SF.Item>
-          <Hds::RichTooltip as |RT|>
-            <RT.Toggle @text="Lorem ipsum dolor" @size={{if (eq size "inherited") undefined size}} @icon="info" />
-            <RT.Bubble>
-              <Shw::Placeholder @text="generic content" @height="40" />
-            </RT.Bubble>
-          </Hds::RichTooltip>
-        </SF.Item>
-        <SF.Item>
-          <Hds::RichTooltip as |RT|>
-            <RT.Toggle
-              @text="Lorem ipsum dolor"
-              @size={{if (eq size "inherited") undefined size}}
-              @icon="info"
-              @iconPosition="leading"
-            />
-            <RT.Bubble>
-              <Shw::Placeholder @text="generic content" @height="40" />
-            </RT.Bubble>
-          </Hds::RichTooltip>
-        </SF.Item>
-      </Shw::Flex>
-    {{/each}}
-  {{/let}}
+  <Shw::Flex @label="Inherited (default)" as |SF|>
+    <SF.Item>
+      <Hds::RichTooltip as |RT|>
+        <RT.Toggle @text="Lorem ipsum dolor" />
+        <RT.Bubble>
+          <Shw::Placeholder @text="generic content" @height="40" />
+        </RT.Bubble>
+      </Hds::RichTooltip>
+    </SF.Item>
+    <SF.Item>
+      <Hds::RichTooltip as |RT|>
+        <RT.Toggle @text="Lorem ipsum dolor" @icon="info" />
+        <RT.Bubble>
+          <Shw::Placeholder @text="generic content" @height="40" />
+        </RT.Bubble>
+      </Hds::RichTooltip>
+    </SF.Item>
+    <SF.Item>
+      <Hds::RichTooltip as |RT|>
+        <RT.Toggle @text="Lorem ipsum dolor" @icon="info" @iconPosition="leading" />
+        <RT.Bubble>
+          <Shw::Placeholder @text="generic content" @height="40" />
+        </RT.Bubble>
+      </Hds::RichTooltip>
+    </SF.Item>
+  </Shw::Flex>
+  {{#each @model.TOGGLE_SIZES as |size|}}
```

Comment:
[thought] since this is going to happen more and more frequently, do you think creating helpers (only for the showcase) that can append/prepend items to an existing array could help? If we were to prepend `inherited` to the `@model.TOGGLE_SIZES` via a `{{#let}}` that defines the `sizes` array, and then leave the code as it was before, would TS complain?

Reply:
TS wouldnt complain if it was done correctly (like we check if the size is inherited and set size to undefined) but I honestly haven't seen any other instance where we would need to prepend an item to an array like this.

Could also remove the inherited example because there are a ton of examples of RichTooltips with size undefined in this page.